### ISSUE TITLE
This package is not working at all

### ISSUE THREAD

**Issue by mdrajib001** [fanday/react-native-aes-encryption#1]
This was just waste of time. No documentation. After I link the package the app even failed to build. 
Please fix the app and kindly add documentation.

**Comment by bitsal** [comment]
@mdrajib001 there is an example
And I can approve it works for both iOS and Android, at least `sha256`, `encrypt` and `decrypt`

**Comment by drriguz** [comment]
@bitsal Yes, I tried several encrypt packages for React Native, this seems to be the only one that works.
